src/blueapi/service/model.py

```diff
@@ -54,10 +54,18 @@ class PlanModel(BlueapiBaseModel):
     """
 
     name: str = Field(description="Name of the plan")
+    description: Optional[str] = Field(description="Docstring of the plan")
+    argument_schema: dict[str, Any] = Field(
```

Comment:
Just for the sake of consistency, we call them parameters rather than arguments throughout the blueapi codebase